sal/class.json-api-site-jetpack.php

```diff
@@ -6,17 +6,42 @@
 // this code runs on Jetpack (.org) sites
 class Jetpack_Site extends Abstract_Jetpack_Site {
 
-	protected function get_mock_option( $name ) {
-		return get_option( 'jetpack_'.$name );
-	}
-
 	protected function get_constant( $name ) {
 		if ( defined( $name) ) {
 			return constant( $name );
 		}
 		return null;
 	}
 
+	protected function main_network_site() {
+		return network_site_url();
+	}
+
+	protected function wp_version() {
+		global $wp_version;
+		return $wp_version;
+	}
+
+	protected function max_upload_size() {
+		return wp_max_upload_size();
+	}
+
+	protected function is_main_network() {
+		return Jetpack::is_multi_network();
+	}
+
+	protected function is_multi_site() {
```

Comment:
it is not necessary to redeclare this one as it's implemented on the parent abstract class